helix-core/src/test/java/org/apache/helix/tools/TestClusterVerifier.java

```diff
@@ -173,15 +179,15 @@ public void testResourceSubset() throws InterruptedException {
 
     // Ensure that this passes even when one resource is down
     _admin.enableInstance(_clusterName, "localhost_12918", false);
-    Thread.sleep(1000);
-    _admin.enableCluster(_clusterName, false);
-    _admin.enableInstance(_clusterName, "localhost_12918", true);
 
     ZkHelixClusterVerifier verifier =
         new BestPossibleExternalViewVerifier.Builder(_clusterName).setZkClient(_gZkClient)
             .setResources(Sets.newHashSet(testDB)).build();
     Assert.assertTrue(verifier.verifyByPolling());
 
+    _admin.enableCluster(_clusterName, false);
```

Comment:
This changes the purpose of this test.
The BestPossibleExternalViewVerifier and StrictMatchExternalViewVerifier should be tested with exactly the same condition.

Why do we need to move these 2 changes to make BestPossibleExternalViewVerifier verifying pass?

Reply:
Fixed. Thanks. Now those checks are together.